enhancements/network/ingress-logging-api.md

```diff
@@ -0,0 +1,364 @@
+---
```

Comment:
If this is for ingress, should the file live in the ingress directory instead of the network directory?

Reply:
Oops, yeah, I see an `ingress/` directory has been added, and this and some other recent enhancements could be moved thither.  Thanks!